packages/core/data/mm/world/stone_tower_temple.yml

```diff
@@ -27,26 +27,24 @@
   dungeon: ST
   exits:
     "Stone Tower Temple Under West Garden": "true"
-    "Stone Tower Temple Center Ledge": "small_keys_st(4) || (small_keys_st(3) && has_mask_zora)"
+    "Stone Tower Temple Center Ledge": "small_keys_st(4) || (small_keys_st(3) && (has_mask_zora || short_hook_anywhere))"
   events:
     STONE_TOWER_WEST_GARDEN_LIGHT: "has_explosives"
 "Stone Tower Temple Under West Garden":
   dungeon: ST
   exits:
     "Stone Tower Temple West Garden": "true"
-    "Stone Tower Temple Under West Garden After Block": "((event(STONE_TOWER_WEST_GARDEN_LIGHT) && has_mirror_shield) || can_use_light_arrows)"
+    "Stone Tower Temple Under West Garden After Block": "(event(STONE_TOWER_WEST_GARDEN_LIGHT) && has_mirror_shield) || can_use_light_arrows || short_hook_anywhere"
   locations:
-    "Stone Tower Temple Under West Garden Ledge Chest": "can_hookshot"
+    "Stone Tower Temple Under West Garden Ledge Chest": "can_hookshot || short_hook_anywhere"
     "Stone Tower Temple Pot Lava Room 1": "true"
     "Stone Tower Temple Pot Lava Room 2": "true"
     "Stone Tower Temple Pot Lava Room 3": "true"
     "Stone Tower Temple Pot Lava Room 4": "true"
 "Stone Tower Temple Under West Garden After Block":
   dungeon: ST
-  exits:
-    "Stone Tower Temple Under West Garden": "true"
```

Comment:
Why was this exit removed?

Reply:
Redundant. Logic already knows you have access to it since it's the only way into here.